src/lib/getSelector.ts

```diff
@@ -34,7 +34,7 @@ export const getSelector = (node: Node | null | undefined, maxLen?: number) => {
           el.classList.value &&
           el.classList.value.trim() &&
           el.classList.value.trim().length
-            ? '.' + el.classList.value.trim().replace(/\s+/g, '.')
+            ? '.' + el.classList.value.trim().split(' ').sort().join('.')
```

Comment:
If we're dropping IE 11 support, we could do something like this, which has the added benefit of deduplicating classes:

```suggestion
            ? '.' + Array.from(el.classList).sort().join('.')
```

I also wonder if there's any reason not to check for el.classList.length, instead of checking the length of the trimmed value.

Reply:
Yes I think this works and allows simplifying it a lot more. Which I've done in https://github.com/GoogleChrome/web-vitals/pull/518/commits/c0481e7ffc8ded7a828bde4ee6914ba51e030216